lua/cmp_path/init.lua

```diff
@@ -153,5 +176,12 @@ source._is_slash_comment = function(_)
   return is_slash_comment and not no_filetype
 end
 
+function source:resolve(completion_item, callback)
+  if completion_item.kind == cmp.lsp.CompletionItemKind.File then
+	completion_item.documentation = try_get_lines(completion_item.path, 10)
+  end
+  callback(completion_item)
```

Comment:
Looks good.

Should we wrap it by markdown fenced codeblock?

Reply:
ok